setup.py

```diff
@@ -28,6 +28,7 @@
         install_requires=[
             "fs~=2.0",
             "google-cloud-storage~=1.0",
+            "numpy~=1.18"
```

Comment:
Why not move the numpy dependency in a separate test_requires?

Reply:
Good point 👍 Will address it in an upcoming separate PR.